src/Compilers/CSharp/Portable/Binder/Binder_InterpolatedString.cs

```diff
@@ -748,7 +755,7 @@ private ImmutableArray<BoundExpression> BindInterpolatedStringParts(BoundUnconve
                     {
                         var boundLiteral = (BoundLiteral)part;
                         Debug.Assert(boundLiteral.ConstantValue != null && boundLiteral.ConstantValue.IsString);
-                        var literalText = ConstantValueUtils.UnescapeInterpolatedStringLiteral(boundLiteral.ConstantValue.StringValue);
+                        var literalText = boundLiteral.ConstantValue.StringValue;
```

Comment:
intentionally removed.  We ensured at creation of hte bound nodes that hte literal constant values are correct.